src/code/__osMalloc.c

```diff
@@ -17,6 +18,8 @@
 OSMesg sArenaLockMsg;
 u32 __osMalloc_FreeBlockTest_Enable;
 
+void __osMallocAddBlock(Arena* arena, void* start, s32 size);
```

Comment:
I'm not sure this is meant to be a private function (not exposed in headers) but this is fine